great_expectations/experimental/datasources/pandas_datasource.py

```diff
@@ -195,13 +195,35 @@ def _validate_batch_request(self, batch_request: BatchRequest) -> None:
 
 _PANDAS_ASSET_MODELS = _generate_pandas_data_asset_models(
     _PandasDataAsset,
-    blacklist=_PANDAS_BLACK_LIST,
+    blacklist=_PANDAS_READER_METHOD_UNSUPPORTED_LIST,
     use_docstring_from_method=True,
     skip_first_param=False,
     type_prefix="pandas",
 )
 
 
+_PANDAS_FILE_TYPE_READER_METHOD_UNSUPPORTED_LIST = (
+    # "read_csv",
+    # "read_json",
+    # "read_excel",
+    # "read_parquet",
+    "read_clipboard",  # not path based
+    # "read_feather",
+    "read_fwf",  # unhandled type
+    "read_gbq",  # not path based
+    # "read_hdf",
+    # "read_html",
+    # "read_orc",
+    # "read_pickle",
+    # "read_sas",  # invalid json schema
+    # "read_spss",
+    "read_sql",  # not path based & type-name conflict
+    "read_sql_query",  # not path based
+    "read_sql_table",  # not path based
+    "read_table",  # type-name conflict
+    # "read_xml",
+)
+
```

Comment:
Why did you move this here?
I think this belongs in `pandas_file_path_datasource.py`

Reply:
Yes -- must be a merge.  I will fix.